### ISSUE TITLE
Incorrect render to PDF using titlepage extension

### ISSUE THREAD

**Issue by dmkaplan2000** [nmfs-opensci/quarto_titlepages#47]
I have already filed this bug with quarto here:

<https://github.com/quarto-dev/quarto-cli/issues/7293>

It is easiest to just look there for the details and solution. Basically, titlepages fails with the newest version of quarto because the file partial `before-body.tex` does not have a new line at the end of the document.

**Comment by eeholmes-uw** [comment]
Interesting. All my pandoc templates do have a newline at the end but when the files are pushed to GitHub, the newlines are eliminated. I will have to figure out how to stop this behavior.

**Comment by eeholmes-uw** [comment]
No. I checked with a local clone. The newlines are all there, including with before-body.tex. I will try to replicate.

**Comment by eeholmes-uw** [comment]
@dmkaplan2000 Thanks for posting. It is a bug with the new Quarto pre-release. I am working with Quarto devs to fix and in the meantime I will add 2 newlines to deal with the pre-release problem.
